src/main.cpp

```diff
@@ -1,15 +1,31 @@
-////////////////////////////////////////////////////////////////////////////
-/**
- *  @file   main.cpp
- *  @date   Fri April 16 2021
- *  @brief  Entry Point of Calculator
- */
-////////////////////////////////////////////////////////////////////////////
-
-#include "Calculator.h"
-
-int main() 
-{
-    // Fill in or change code here as necessary.  See also Calculator.cpp
-    std::cout << "Welcome to the Calculator C++ learning project." << std::endl;
+////////////////////////////////////////////////////////////////////////////
+/**
+ *  @file   main.cpp
+ *  @date   Fri April 16 2021
+ *  @brief  Entry Point of Calculator
+ */
+////////////////////////////////////////////////////////////////////////////
+
+#include <iostream>
+#include <vector>
+
+#include "Symbol.h"
+
+int main() 
+{
+    using namespace std;

```

Comment:
While `using namespace x` is allowed in small scopes, it is still generally discourages, especially `std`